modules/creating-serverless-apps-kn.adoc

```diff
@@ -1,9 +1,5 @@
-// Module included in the following assemblies:
-//
-// serverless/knative_serving/serverless-applications.adoc
-
 [id="creating-serverless-apps-kn_{context}"]
-= Creating serverless applications using the kn CLI
+= Creating serverless applications using the Knative CLI
```

Comment:
these should all be `by using` per the IBM style Guide. Without the _by_, it could be assumed that you are creating an application that is using the Knative CLI, not that you are using the CLI to create the application,

Reply:
Updating these, thanks